src/webpackPages.js

```diff
@@ -101,7 +101,10 @@ const webpackPages = (globalOptions) => {
     }
 
     outputFiles = { }
-    const promises = Object.keys(files).map(function (key) {
+    const promises = Object.keys(files)
+      // Filter out files that have a trailing slash. (will become something like "/foobarbaz/.html", which can't be build)
+      .filter(file => !/\/.html$/.test(file))
```

Comment:
It would be worth giving some kind of feedback when this happens so that any invalid configurations aren't just lost to the void